operations/services.md

```diff
@@ -227,16 +250,16 @@ Returns selected availability and occupancy metrics of a bookable service for a
     {
       "ResourceCategoryId": "d1801d11-fe8d-404b-a26f-af170189605a",
       "Metrics": {
-        "OutOfOrderBlocks": [0, 1, 0, 0, 1],
-        "PublicAvailabilityAdjustment": [7, 5, 4, 3, 4],
-        "OtherServiceReservationCount": [0, 0, 1, 0, 0],
-        "Occupied": [7, 5, 4, 3, 4],
-        "ConfirmedReservations": [7, 5, 4, 3, 4],
-        "OptionalReservations": [0, 2, 0, 0, 1],
-        "BlockAvailability": [0, 0, 1, 0, 0],
-        "AllocatedBlockAvailability": [0, 0, 0, 1, 0],
-        "UsableResources": [8, 8, 8, 8, 8],
-        "ActiveResources": [8, 8, 8, 8, 8]
+        "OutOfOrderBlocks": [ 0, 1, 0, 0, 1 ],
```

Comment:
nitpick: Are these spaces around intentional? 🤔

Reply:
No, fixed it.